cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/src/tests/snowbridge_v2_outbound.rs

```diff
@@ -1008,7 +1008,10 @@ fn export_message_from_asset_hub_to_ethereum_is_banned_when_set_operating_mode()
 				bx!(xcm),
 				Weight::from(EXECUTION_WEIGHT),
 			),
-			pallet_xcm::Error::<Runtime>::LocalExecutionIncomplete
+			pallet_xcm::Error::<Runtime>::LocalExecutionIncompleteWithError {
+				error: XcmError::NoDeal.into(),
+				index: 0
```

Comment:
Yes, it shall be `Unroutable`.

```suggestion
				error: XcmError::Unroutable.into(),
				index: 2
```